electron/legendary_utils/library.ts

```diff
@@ -272,6 +272,10 @@ class Library {
     const gameBoxTall = is_game ?
       keyImages.filter(({ type }: KeyImage) => type === 'DieselGameBoxTall' )[0] :
       gameBox
+    
+    const gameBoxStore = is_game ?
+      keyImages.filter(({ type }: KeyImage) => type === 'dieselStoreFrontTall' )[0] :
```

Comment:
In the JSON in #386 dieselStoreFrontTall is actually uppercase DieselStoreFrontTall